### PR TITLE
kmplayer Cheat Sheet

### PR DESCRIPTION
**Pull request by mesooryanarayananb**
**What does your Instant Answer do?**

Give an instant answer to a searched Keyword

**What problem does your Instant Answer solve (Why is it better than organic links)?**

It gives cheatsheets for KMplayer 

**What is the data source for your Instant Answer? (Provide a link if possible)**

KMplayer Software , The data source is the software , I reviewed each menus in it.

**Why did you choose this data source?**

Because KMplayer is an widely used Software . I hope it will be helpful for many users

**Are there any other alternative (better) data sources?**

I dont no , lets duck it !!!

**What are some example queries that trigger this Instant Answer?**
Kmplayer , KMplayer cheatsheet

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
People who use KMplayer to see movies and videos

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No ,I think

**Which existing Instant Answers will this one supercede/overlap with?**
none

**Are you having any problems? Do you need our help with anything?**
I want to start tech events about duckduckgo in my college , so maybe some interested students will create cheatsheets and instantanswer for duckduckgo . So can I do that ? If yes ! Is there any procedures ??

**Where did you hear about DuckDuckHack? (For first time contributors)**
Hackton Event Kochi

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

## Checklist
Please place an 'X' where appropriate.

```
- [] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [x] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```
screenshot 41

### CODE REVIEW

**Comment by alexander95015** [comment]
@mesooryanarayananb For cheat sheets, only the json file is needed. The Perl file and the test file are not necessary for cheat sheets.

**Comment by mesooryanarayananb** [comment]
@alexander95015  Should I remove it ?

**Comment by alexander95015** [comment]
@mesooryanarayananb Probably yes. It looks like those two files are causing the check to fail.

**Comment by mesooryanarayananb** [comment]
@alexander95015 Could you please help me with it ?

**Comment by alexander95015** [comment]
@mesooryanarayananb Just delete the files and push the commits to your fork and it'll show up here.

**Comment by mesooryanarayananb** [comment]
Am new here !

File: t/IsAwesome/mesooryanarayananb.t [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env perl
```
Comment (gautamkrishnar) [inline t/IsAwesome/mesooryanarayananb.t]: Please delete this file

File: lib/DDG/Goodie/IsAwesome/mesooryanarayananb.pm [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package DDG::Goodie::IsAwesome::mesooryanarayananb;
```
Comment (gautamkrishnar) [inline lib/DDG/Goodie/IsAwesome/mesooryanarayananb.pm]: Delete this too

**Comment by gautamkrishnar** [comment]
@mesooryanarayananb I will help you :+1:

**Comment by gautamkrishnar** [comment]
pls close the pull request. We can create a new one

**Comment by gautamkrishnar** [comment]
close this one!

**Comment by mesooryanarayananb** [comment]
Moved to:
https://github.com/duckduckgo/zeroclickinfo-goodies/pull/1510

**Pull request closed without merging**